Mathlib/CategoryTheory/Sites/Coverage.lean

```diff
@@ -285,6 +285,17 @@ theorem toGrothendieck_eq_sInf (K : Coverage C) : toGrothendieck _ K =
     intro X S hS
     apply saturate.of _ _ hS
 
+/-- The union of two coverages is a coverage. -/
+@[simps]
+def union (x y : Coverage C) : Coverage C where
```

Comment:
Would it make sense to make this a `Sup` instance? Is `SemilatticeSup (Coverage C)` true?

Reply:
Done!